book/forms.rst

```diff
@@ -1021,6 +1021,29 @@ to the ``form()`` or the ``form_start()`` helper:
     a PUT, PATCH or DELETE request. Read the cookbook chapter
     ":doc:`/cookbook/routing/method_parameters`" for more information.
 
+.. note::
+
+  Now that the ``createForm()`` method requires the FQCN, you can not longer pass in ``__construct()`` arguments. To get 
```

Comment:
I agree, I will wait to see what the other team members have to say before creating a pull request for previous version documentations.